crates/bevy_ui/src/ui_node.rs

```diff
@@ -1819,10 +1821,15 @@ impl Outline {
 }
 
 /// The 2D texture displayed for this UI node
-#[derive(Component, Clone, Debug, Reflect, Default)]
+#[derive(Component, Clone, Debug, Reflect)]
 #[reflect(Component, Default)]
 pub struct UiImage {
-    /// The tint color used to draw the image
+    /// The tint color used to draw the image.
+    ///
+    /// # Warning
+    ///
+    /// This defaults to transparent white,
```

Comment:
I'm a little confused with the word transparent here, as this can be seen as rgba(1, 1, 1, 0) (white but transparent), but in reality its literally white.
And in fact, now both the Color::default and UiImage::default will lead to white, so the image will show up. 
I guess we can warn the user about the fact that the Color here is multiplied with the img, so white will be the original colors and transparent will be nothing.

Reply:
Now I understand the part about transparent white, though I need to test how this will go in some example. But how about we have the explanation here so the user know the implication of this color as a field itself?

Reply:
> I'm a little confused with the word transparent here, as this can be seen as rgba(1, 1, 1, 0) (white but transparent), but in reality its literally white.

I'm quite confused by this comment.

Reply:
> > I'm a little confused with the word transparent here, as this can be seen as rgba(1, 1, 1, 0) (white but transparent), but in reality its literally white.
> 
> I'm quite confused by this comment.

Yeah, this was regarding me not seeing that this was transparent white (comment below). You can ignore that part now.